ebb_lint/flake8.py

```diff
@@ -273,6 +273,7 @@ def _check_tree(self, tree):
                 if not pattern.match(node, results):
                     continue
                 for k in extra.get('comments_for', ()):
+                    # XXX: this doesn't use `k` for finding the node
```

Comment:
`k` is supposed to name a specific node, but it isn't used for selecting a node when adding things to the results. I'll change the comment to make that clearer.